handlers/facebookapp/facebookapp.go

```diff
@@ -56,13 +59,25 @@ const (
 	payloadKey    = "payload"
 )
 
+var waStatusMapping = map[string]courier.MsgStatusValue{
+	"sent":      courier.MsgSent,
+	"delivered": courier.MsgDelivered,
+	"read":      courier.MsgDelivered,
+	"failed":    courier.MsgFailed,
+}
+
+var waIgnoreStatuses = map[string]bool{
+	"deleted": true,
+}
+
 func newHandler(channelType courier.ChannelType, name string, useUUIDRoutes bool) courier.ChannelHandler {
 	return &handler{handlers.NewBaseHandlerWithParams(channelType, name, useUUIDRoutes)}
 }
 
 func init() {
 	courier.RegisterHandler(newHandler("IG", "Instagram", false))
 	courier.RegisterHandler(newHandler("FBA", "Facebook", false))
+	courier.RegisterHandler(newHandler("CWA", "Cloud API WhatsApp", false))
```

Comment:
how about we call this `WAC` / `WhatsApp Cloud` in code so that lexically this channel type will appear with the existing `WA`.. and if I'm looking for a WhatsApp channel type, I'm probably going to start looking under `W`